itests/org.openhab.io.hueemulation.tests/src/main/java/org/openhab/io/hueemulation/internal/HueEmulationServiceOSGiTest.java

```diff
@@ -47,7 +48,7 @@
  */
 public class HueEmulationServiceOSGiTest extends JavaOSGiTest {
     private HueEmulationService hueService;
-    VolatileStorageService volatileStorageService = new VolatileStorageService();
+    private VolatileStorageService volatileStorageService = new VolatileStorageService();
 
```

Comment:
We cannot retrieve the `UpnpService` in this test, because of that it fails. I can inject a mock but then the result does not make sense anymore, it reports port 0.